influx/query.go

```diff
@@ -0,0 +1,45 @@
+package influx
+
+import (
+	"context"
+	"crypto/x509"
+	"fmt"
+
+	"github.com/apache/arrow/go/v12/arrow/flight"
+	"github.com/apache/arrow/go/v12/arrow/flight/flightsql"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/metadata"
+)
+
+func (c *Client) Query(ctx context.Context, bucket string, query string, queryParams interface{}) (*flight.Reader, error) {

```

Comment:
Please rename to database:
```suggestion
func (c *Client) Query(ctx context.Context, database string, query string, queryParams interface{}) (*flight.Reader, error) {

```